src/main/java/io/vertx/core/http/impl/HttpClientImpl.java

```diff
@@ -128,6 +128,7 @@
   private final boolean keepAlive;
   private final boolean pipelining;
   private final CloseFuture closeFuture;
+  private final ContextInternal timerContext;
```

Comment:
we don't need to keep a reference on it as the timer task can simply call vertx.setTimer that will schedule the call on the same context

Reply:
done